.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "openQA Single Instance",
+    "image": "registry.opensuse.org/devel/openqa/containers15.5/openqa-single-instance",
+    "runArgs": [ "--privileged", "--device", "/dev/kvm", "--entrypoint", "bash" ],
+    "postCreateCommand": "chown root:kvm /dev/kvm",
+    "containerEnv": {
+      "VNCPORT_OFFSET": "100"
+    },
+    "postStartCommand": "/usr/share/openqa/script/openqa-bootstrap >/var/log/openqa-bootstrap.log 2>&1"
```

Comment:
I found an example which spawns a node application https://github.com/blackgirlbytes/codespaces-forward-port

The web app responds to ^C and I see a "bash" tab.

![image](https://github.com/os-autoinst/os-autoinst-distri-example/assets/1204189/9c8ac009-580f-422a-aef3-30c61e962d48)

This is clear enough for me. I'm wondering why we don't see this in the openQA setup

Reply:
Also in this node example app it tells me what it's doing and where the server is running (it's not fully visible in the screenshot).

Reply:
I guess [it's using](https://github.com/blackgirlbytes/codespaces-forward-port/blob/main/.devcontainer/devcontainer.json#L9) `postAttachCommand` and maybe that makes a difference?

Reply:
maybe I need to use `postAttachCommand` instead

Reply:
Nice! Much better now. Thank you for making those tweaks 🥰